superset-frontend/src/SqlLab/components/SqlEditor/index.jsx

```diff
@@ -663,6 +663,11 @@ class SqlEditor extends React.PureComponent {
       </Menu>
     );
 
+    const queryWithColumns = () => ({
+      ...qe,
+      columns: this.props.latestQuery?.results?.columns || [],
+    });
+    console.log('findme SqlEditor', qe, queryWithColumns());
```

Comment:
remove this

Reply:
This was just debugging, it wasn't ready for review yet. I pushed up a "clean up" commit [`here`](https://github.com/apache/superset/pull/20975/commits/9f348bddec74224a808ac5ba88c7883e42f78c6a).